tests/framework/ROM/TimeSeries/MultiResolutionTSA/multiYearDWT.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/ROM/TimeSeries/SyntheticHistory.Fourier</name>
+    <author>talbpaul</author>
+    <created>2021-01-08</created>
+    <classesTested>SupervisedLearning.SyntheticHistory,TSA.Fourier</classesTested>
+    <description>
+      Tests the SyntheticHistory ROM using only the Fourier TimeSeriesAnalyzer algorithm.
```

Comment:
I would suggest to add descriptions to reflect multi-resolution TSA

Reply:
fixed